docs/content/getting-started/create-new-project.mdx

```diff
@@ -129,14 +129,16 @@ Inside of the `my_dagster_project/` directory, the following files and directori
       </td>
     </tr>
     <tr>
-      <td>my_dagster_project/assets/</td>
+      <td>my_dagster_project/assets.py</td>
       <td>
-        A Python package that contains all the{" "}
-        <a href="/concepts/assets/software-defined-assets">assets</a>. It is
-        helpful to include all assets in a subpackage so that you can use{" "}
-        <PyObject object="load_assets_from_package_module" /> to load assets
-        into your project, rather than needing to add assets to the
-        project every time you define one.
+        A Python module that contains {" "}
+        <a href="/concepts/assets/software-defined-assets">Software-Defined Assets</a>.
+        <br />
+        <br />
+        Note: As your project grows, we recommend organizing assets in subpackages, e.g. <code>my_dagster_project/assets/analytics.py</code>, so that you can use{" "}
+        <PyObject object="load_assets_from_package_module" /> to load assets into your project and
+        group them based on the folder structure, rather than needing to manually group assets every time you define one.
```

Comment:
> rather than needing to manually group assets every time you define one.

`group` as the verb here strikes me a bit weird. When I define a new asset, I don't think about grouping them after the fact, but more about they're included for loading. Does that make sense?

Reply:
reword it to "manually add assets to the top-level definitions"